Source/ACE/Entity/Player.cs

```diff
@@ -1629,6 +1782,25 @@ public void UpdatePlayerBurden()
         }
 
         /// <summary>
+        /// Tracks Interacive world object you are have interacted with recently.
+        /// </summary>
+        /// <param name="worldObject"></param>
+        public void TrackInteractiveObjects(List<WorldObject> worldObjects)
+        {
+            // todo: figure out a way to expire objects.. objects clearly not in range of interaction /etc
+            new ActionChain(this, () =>
```

Comment:
As implemented this is only called from contexts on the player, and shouldn't be in an ActionChain.  (Due to action timing delay you may actually create bugs putting it in one that you won't notice for a while, and will be hard to debug :-/)